Source/ThirdParty/libwebrtc/Configurations/libwebrtc.xcconfig

```diff
@@ -83,4 +83,8 @@ EXCLUDED_SOURCE_FILE_NAMES[sdk=xrsimulator*][arch=arm64*] = $(EXCLUDED_SOURCE_FI
 EXCLUDED_SOURCE_FILE_NAMES[sdk=macosx*] = $(EXCLUDED_SOURCE_FILE_NAMES_macosx);
 EXCLUDED_SOURCE_FILE_NAMES[sdk=macosx*][arch=arm64*] = $(EXCLUDED_SOURCE_FILE_NAMES_macosx) $(EXCLUDED_SOURCE_FILE_NAMES_arm);
 
-OTHER_LDFLAGS = $(inherited) $(SOURCE_VERSION_LDFLAGS) $(WEBRTC_LDFLAGS_ENABLE_LIBFUZZER_$(ENABLE_LIBFUZZER)) -framework CoreGraphics -allowable_client WebCore -allowable_client WebCoreTestSupport -allowable_client WebKit;
+OTHER_LDFLAGS = $(inherited) $(SOURCE_VERSION_LDFLAGS) $(WEBRTC_LDFLAGS_ENABLE_LIBFUZZER_$(ENABLE_LIBFUZZER)) -framework CoreGraphics $(WEBRTC_ALLOWABLE_CLIENTS);
+
+// Allow fuzzers to link to libwebrtc.dylib.
+WEBRTC_ALLOWABLE_CLIENTS = $(WEBRTC_ALLOWABLE_CLIENTS_$(WK_NOT_$(ENABLE_LIBFUZZER)));
```

Comment:
This double negative is quite confusing, it took me a couple minutes to understand how this fixes the issue. It would be better to have a more descriptive name and no negation (say, `WEBRTC_ALLOWABLE_CLIENTS_LIBFUZZER_NO`).